.github/workflows/pr.yaml

```
@@ -795,6 +812,32 @@ jobs:
+    uses: ./.github/workflows/shared-test.yml
+    with:
+      dd_flaky_retry_enabled: 'false'
+      test_suite_name: vllm
+      test_type: GB200 Test
+      # 2x GB200 ARM64 lane; validates the nightly single- and dual-GPU test pool.
+      arm_runner: aws-dev-01-tester-arm-gpu-v2
+      target_tag_plain: ${{ needs.vllm-build.outputs.target_tag_plain }}
+      cuda_version: '["13.0"]'
+      platform: '["arm64"]'
+      models_dir: /models
+      enable_coverage: true
```

actionlint (1.7.12)

[error] input "enable_coverage" is typed as string by reusable workflow "./.github/workflows/shared-test.yml". bool value cannot be assigned

(expression)